demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -29,6 +31,56 @@
 
 json = JSON_Handler()
 
+CLASS_BY_FILE_TYPE = {FileType.INTEGRATION: Integration, FileType.SCRIPT: Script, FileType.PLAYBOOK: Playbook}
+
+
+def commands_to_deprecate_set(commands: list) -> set:
+    """return a set of the deprecated commands only"""
+    res_set = set()
+    for command in commands:
+        if command.get('deprecated'):
+            res_set.add(command.get('name'))
+
+    return res_set
+
+
+def get_yml_objects(path: str, file_type) -> Tuple[Any, Union[Integration, Script, Playbook]]:
+    """
+    Generate yml files from master and from the current branch
+    Args:
+        path: The requested file path
+        file_type: the file type
+
+    Returns:
+        Two YML objects, the first of the yml at master (old yml) and the second from the current branch (new yml)
+    """
+    old_yml_obj = get_remote_file(path)
+    new_yml_obj = CLASS_BY_FILE_TYPE[file_type](path)
+
+    return old_yml_obj, new_yml_obj
+
+
+def get_deprecated_rn(path: str, file_type):
+    """Generate rn for deprecated items"""
+    old_yml, new_yml = get_yml_objects(path, file_type)
```

Comment:
do we care that these are yml objects? content_items seems a bit more clear to me